pylot/perception/tracking/object_tracker_operator.py

```diff
@@ -73,12 +76,17 @@ def on_frame_msg(self, msg, obstacle_tracking_stream):
         if self._ready_to_update:
             self.__track_bboxes_on_frame(camera_frame, msg.timestamp, False,
                                          obstacle_tracking_stream)
+            runtime = (time.time() - start_time) * 1000
```

Comment:
We've moved away from manually logging runtimes into csv. Instead, you can add `@erdos.profile_method()` to profile a method or you can use `with erdos.profile()` to get the runtime of a block of code. The runtime info is saved in the file set by FLAGS.profile_file_name, and can be visualized in Chrome.
Can you please use erdos.profile instead of the old way of logging runtime?